lib/server/routes/users.js

```diff
@@ -26,6 +27,19 @@ function UsersRouter(options) {
   Router.apply(this, arguments);
 
   this._verify = authenticate(this.storage);
+
+  this._defaults = {
+    lookup: function(req) {
+      return [req.user, req.connection.remoteAddress]
```

Comment:
`req.user` isn't going to populated yet on many of these, though I don't think this property is being used?